src/main/java/core/basesyntax/db/FruitStorage.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.db;
+
+import core.basesyntax.models.Fruit;
+import java.util.Map;
+
+public record FruitStorage(Map<Fruit, Integer> storage) {
```

Comment:
`record` is usually used in the DTO classes and makes them immutable as far as I am aware, in this case `Storage` acts the role of the DB, or DAO, or Service, so let's make it a usual calss with `public/private static final Map`